app/models/concerns/searchable.rb

```diff
@@ -23,9 +23,9 @@ module Searchable
     end
 
     # shorthand methods
-    def reindex
+    def reindex(refresh = false)
       __elasticsearch__.index_document
-      refresh_index
+      refresh_index if refresh
```

Comment:
Also decided not to refresh the index on create/update.  But it does refresh on destroy.  Hoping that will speed up those operations a little bit.  (They're async processed anyways, so user really shouldn't notice).